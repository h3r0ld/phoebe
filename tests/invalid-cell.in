start map.txt
hardworking-little-robot-random off
move S
end-turn
quit
