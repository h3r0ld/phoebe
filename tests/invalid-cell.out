Player 0: Cell(22,8) Speed: 2 CanChangeDirection: True
Player 0: New Position: Cell(24, 8)
Player 0: Distance - 2; Player 1: Distance - 0; Winner: Player 1